checks/contributors.go

```diff
@@ -38,6 +43,28 @@ func Contributors(c checker.Checker) checker.CheckResult {
 			if err != nil {
 				return checker.RetryResult(err)
 			}
+			// TODO - Figure out the real API for getting user orgs
+			url := fmt.Sprintf("https://api.github.com/users/%s/orgs", contrib.GetLogin())
+			resp, e := c.HttpClient.Get(url)
+
+			if e != nil {
+				c.Logf("unable to get org members for %s", contrib.GetLogin())
+			} else {
+				var orgs orgnaizations
+
+				err = json.NewDecoder(resp.Body).Decode(&orgs)
+
+				if err != nil {
+					c.Logf("unable to get  decode json for for %s org", contrib.GetLogin())
```

Comment:
nit: 2 spaces after get. two repeated fors, also it is user for %s user or just %s.

Reply:
Took care of it.